package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "blackbaud-skyux-lib-restricted-view",
```

Comment:
Will the name of the NPM package be `@blackbaud/skyux-lib-restricted-view`? If so, it should be added here. (Might need to change the name of the repo, too?)

Reply:
Can the library name be updated? Bobby set it up for us, but I've never gone through that process.

Reply:
Good point. I updated the repo name and hooked up Travis!